index.html

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
I have nothing against markdown / am fine with using markdown, I'm just not as flexible with it as with HTML. 

It's up to you :-).